### DISCUSSION TITLE
House ware Github Octernship

### DISCUSSION THREAD

**Discussion by ishaan812** [community/community#51891] [GitHub Education]
I made a submission for the housewares project. Im not sure how the process is going to work next, will my project even get reviewed I spent a lot of time on it.

**Comment by samus-aran** [comment]
Hi @ishaan812 

We would recommend students working on Octernship projects to reach out to the project’s maintainers to ask for clarification on the proposed tasks or identify any issues they are experiencing. All project’s may have different preferred methods of communication but here are a few places to check where best to raise inquiry:

- Octernships Project’s README - This may include instructions on how and where to raise an inquiry.
- Octernships Project’s Discussions - Not all projects’ repos have this enabled. We encourage students to first search in case your question has already been answered.
- Octernships Project’s Issues - We encourage students to first search in case your question has already been answered. We also encourage students to follow instructions on the issue template(s) on how to create a suitable issue if provided.

Hope this helps 👍

**Comment by Kanika637** [comment]
I also did HousewareHQ's assignment, but haven't heard from them for a good time now. I also mailed the mentor but no response.

**Reply by ishaan812** [reply]
Yep dont know what to do either, i think houseware octernships have been taken off as a whole. The whole octernships thing has been a shitshow.
